lib/all-badge-examples.js

```diff
@@ -966,7 +966,14 @@ const allBadgeExamples = [
       {
         title: 'JetBrains Plugins',
         previewUri: '/jetbrains/plugin/v/9630-a8translate.svg'
-      }
+      },
+      {
+        title: 'PHP from Packagist',
+        previewUri: '/php/v/packagist/symfony/symfony/dev-master.svg',
```

Comment:
I'd suggest we make this `/packagist/php-v/symfony/symfony` instead. It's similar to the way we handle PyPI and CocoaPods.

Especially if there are going to be more than one way to get a package's PHP version, it will be clearer to put the vendor first.

Reply:
fixed